tedana/model/fit.py

```diff
@@ -188,6 +199,19 @@ def fitmodels_direct(catd, mmix, mask, t2s, t2s_full, tes, combmode, ref_img,
         kappas[i_comp] = np.average(F_R2, weights=norm_weights)
         rhos[i_comp] = np.average(F_S0, weights=norm_weights)
 
+    if verbose:
```

Comment:
It looks like you're writing out _new_ files with `verbose` enabled ? That is, things that had never been written out before, in any previous version ?

Reply:
In order to binderize the walkthrough notebooks (from #149), we need these additional files. I figured that I would roll that into this PR since we're adding `verbose`, but I can push it to another PR.

Reply:
Do you mind if we push it off ? I'd like to walk through that whole binderizing process, and I think I'll get confused having it in several different PRs.

Reply:
Done.